ECOSYSTEM.md

```diff
@@ -0,0 +1,86 @@
+
+[kitchen-qemu][qemu] | [kitchen-cfengine][cfengine] | [kitchen-cloudformation][cloudformation] | [kitchen-wpar][wpar] | [kitchen-powervc][powervc]
+---- | ---- | ---- | ---- | ---- | ----
+[![Status](https://travis-ci.org/esmil/kitchen-qemu-qemu.svg?branch=master)](https://travis-ci.org/esmil/kitchen-qemu)| [![Status](https://travis-ci.org/nmische/kitchen-cfengine.svg?branch=master)](https://travis-ci.org/nmische/kitchen-cfengine) | [![Status](https://travis-ci.org/neillturner/kitchen-cloudformation.svg?branch=master)](https://travis-ci.org/neillturner/kitchen-cloudformation) | [![Status](https://travis-ci.org/adejoux/kitchen-wpar.svg?branch=master)](https://travis-ci.org/adejoux/kitchen-wpar) |  [![Status](https://travis-ci.org/chmod666org/kitchen-powervc.svg?branch=master)](https://travis-ci.org/chmod666org/kitchen-powervc)
+[![Gem Version](https://badge.fury.io/rb/kitchen-qemu.svg)](http://badge.fury.io/rb/kitchen-qemu) | [![Gem Version](https://badge.fury.io/rb/kitchen-cfengine.svg)](http://badge.fury.io/rb/kitchen-cfengine) | [![Gem Version](https://badge.fury.io/rb/kitchen-cloudformation.svg)](http://badge.fury.io/rb/kitchen-cloudformation) | [![Gem Version](https://badge.fury.io/rb/kitchen-wpar.svg)](http://badge.fury.io/rb/kitchen-wpar) | [![Gem Version](https://badge.fury.io/rb/kitchen-powervc.svg)](http://badge.fury.io/rb/kitchen-powervc)
+
+
+[chefpartners]: https://github.com/chef-partners/
+[ec2]: https://github.com/test-kitchen/kitchen-ec2
+[do]: https://github.com/test-kitchen/kitchen-digitalocean
+[open]: https://github.com/test-kitchen/kitchen-openstack
+[rs]: https://github.com/test-kitchen/kitchen-rackspace
+[google]: https://github.com/test-kitchen/kitchen-google
+[vagrant]: https://github.com/test-kitchen/kitchen-vagrant
+[dsc]: https://github.com/test-kitchen/kitchen-dsc
+[pester]: https://github.com/test-kitchen/kitchen-pester
+[joyent]: https://github.com/test-kitchen/kitchen-joyent
+[opennebula]: https://github.com/test-kitchen/kitchen-opennebula
+[hyperv]: https://github.com/test-kitchen/kitchen-hyperv
+[cloudstack]: https://github.com/test-kitchen/kitchen-cloudstack
+[vra]: https://github.com/chef-partners/kitchen-vra
+[vro]: https://github.com/chef-partners/kitchen-vro
+[vcair]: https://github.com/chef-partners/kitchen-vcair
+[oracle]: https://github.com/chef-partners/kitchen-oraclecloud
+[sadpanda]:  https://github.com/chef-partners/kitchen-ssh-cisco
+[sync]:  https://github.com/coderanger/kitchen-sync
+[ansible]: https://github.com/neillturner/kitchen-ansible
+[puppet]: https://github.com/neillturner/kitchen-puppet
+[salt]: https://github.com/kitchen-salt/kitchen-salt
+[dokken]: https://github.com/someara/kitchen-dokken
+[lxc]: https://github.com/chrisroberts/kitchen-lxc
+[nodes]: https://github.com/mwrock/kitchen-nodes
+[vsphere]: https://github.com/mwrock/kitchen-vsphere
```

Comment:
This gem is the deprecated driver. The "official" one is bundled with the provisioner at https://github.com/CenturyLinkCloud/chef-provisioning-vsphere